test/test_blocks.py

```diff
@@ -351,18 +352,19 @@ async def test_run_without_launching(self):
 
```

Comment:
This test case was introduced in https://github.com/gradio-app/gradio/pull/3089, and became unnecessary in https://github.com/gradio-app/gradio/pull/5267 ?

Reply:
Yea I think we can delete this

Reply:
Thanks!

Reply:
Thanks, deleted 👍